src/bin/pgcopydb/pgsql.h

```diff
@@ -568,8 +568,8 @@ bool pgsql_configuration_exists(PGSQL *pgsql, const char *setconfig, bool *exist
 
 bool pgsql_table_exists(PGSQL *pgsql,
 						uint32_t oid,
-						const char *relname,
```

Comment:
I did not see a change in callsites, dis you check if anything is needed there?

Reply:
There is nothing wrong with the calling sides.

Reply:
I see it's only in the header file. Missed that parts in my previous round of review. Kind of makes sense. I wish the compiler would complain, maybe gcc does but not clang or something.